libs/angular-components/styles/vertical-spacing.scss

```diff
@@ -0,0 +1,19 @@
+/* Used only in Vertical spacing components */
+
+.tedi-vertical-spacing {
+  > * {
+    margin-bottom: var(--vertical-spacing-internal, 0);
```

Comment:
maybe use the underscore naming convention for consistency `--_vertical-spacing`